cmd/web/main.go

```diff
@@ -503,28 +505,54 @@ WHERE p.id = ?1`
 	if lang == "" || !safe {
 		lang, _ = enry.GetLanguageByContent(fname, []byte(data))
 	}
-
 	var rawHTML *template.HTML
 
 	// XXX(Xe): HACK around https://github.com/go-enry/go-enry/pull/154
 	if filepath.Ext(fname) == ".markdown" {
 		lang = "Markdown"
 	}
 
+	// For whatever reason go-enry can't correctly match org files?
+	if filepath.Ext(fname) == ".org" {
+		lang = "Org"
+	}
+
```

Comment:
This will likely be resolved by #20 as enry also supports this. 
https://github.com/go-enry/go-enry/blob/93616cf55babb3e8d3b5ee143033c53511c7ce88/data/extension.go#L770